mavenembedder-workitem/src/main/java/org/jbpm/process/workitem/mavenembedder/MavenEmbedderCommand.java

```diff
@@ -52,4 +53,19 @@ public ExecutionResults execute(CommandContext ctx) throws Exception {
             throw new RuntimeException(e.getMessage());
         }
     }
+    
+    
+    protected Object getData(CommandContext ctx, String name) {
+    	WorkItem workItem = (WorkItem) ctx.getData("workItem");
+    	Object data = ctx.getData(name);
```

Comment:
Maybe just an improvement - we can leave this as null since this is also called in the second if block.

Reply:
makes sense, updated